### PR TITLE
Added --show-labels

### LINKED ISSUE #14778: Document --show-labels
**What would you like to be added**
Add --show-labels options as part of [kubectl cheatsheet](https://kubernetes.io/docs/reference/kubectl/cheatsheet/)
Currently documentation has length way to find it out 
`# Show labels for all pods (or any other Kubernetes object that supports labelling) is # Also uses "jq"
for item in $( kubectl get pod --output=name); do printf "Labels for %s\n" "$item" | grep --color -E '[^/]+$' && kubectl get "$item" --output=json | jq -r -S '.metadata.labels | to_entries | .[] | " \(.key)=\(.value)"' 2>/dev/null; printf "\n"; done
`

**Why is this needed**
Quick reference to get to the Kubernetes object labels

### PR DESCRIPTION
**Pull request by dpandhi-git**
Added --show-labels as per #14778

### CODE REVIEW

**Comment by tengqm** [comment]
The 'OR' added inside a code snippet is breaking the flow.

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -182,6 +182,10 @@ echo $(kubectl get pods --selector=$sel --output=jsonpath={.items..metadata.name
 # Also uses "jq"
 for item in $( kubectl get pod --output=name); do printf "Labels for %s\n" "$item" | grep --color -E '[^/]+$' && kubectl get "$item" --output=json | jq -r -S '.metadata.labels | to_entries | .[] | " \(.key)=\(.value)"' 2>/dev/null; printf "\n"; done
 
+OR
```
Comment (hpandeycodeit) [inline content/en/docs/reference/kubectl/cheatsheet.md]: @dpandhi-git This "OR" can be put in the comment

Comment (dpandhi-git) [reply content/en/docs/reference/kubectl/cheatsheet.md]: Thank you for the comments @tengqm @hpandeycodeit

**Comment by hpandeycodeit** [comment]
Thanks @dpandhi-git! 

/lgtm

**Comment by hpandeycodeit** [comment]
```
for item in $( kubectl get pod --output=name); do printf "Labels for %s\n" "$item" | grep --color -E '[^/]+$' && kubectl get "$item" --output=json | jq -r -S '.metadata.labels | to_entries | .[] | " \(.key)=\(.value)"' 2>/dev/null; printf "\n"; done

```

The above script uses `jq` command and this needs to be installed in order to run this script successfully. 

 ```kubectl get pods --show-labels ```  does that job in one line. 

So I think the above script can be removed.

**Pull request merged**